src/backend/base/langflow/components/brightdata/brightdata_web_scraper.py

```
@@ -0,0 +1,189 @@
+                    "error": error_msg,
+                    "status_code": response.status_code
+                }
+                
+                return Data(
+                    text=error_msg,
+                    data=self._metadata
+                )
+                
+        except requests.exceptions.Timeout:
+            error_msg = f"Request timed out after {self.timeout} seconds"
+            return self._create_error_data(url if 'url' in locals() else "unknown", error_msg)
```

Ruff (0.11.9)

Single quotes found but double quotes preferred

Replace single quotes with double quotes

(Q000)